---
title: "Comparison of tet(X4)-containing contigs assembled from metagenomic sequencing data with plasmid sequences of isolates from a cohort of healthy subjects"
collection: publications
category: manuscripts
permalink: /publication/tetX4-mgs
excerpt: 'We demonstrate the effectiveness of metagenomic sequencing in detecting the tigecycline resistance gene, <i>tet</i>(X4) in faecal samples from a cohort of healthy human subjects. Furthermore, we also discovered a frequently occurring multidrug resistance cassette inclusive of a recently discovered tylosin resistance gene.'
date: 2024-03-05
venue: '<a href="https://doi.org/10.1128/spectrum.03969-23">Microbiology Spectrum</a>'
slidesurl: # 'http://academicpages.github.io/files/slides1.pdf'
paperurl: 'https://shuaneryx.github.io/files/ding-et-al-2024-comparison-of-tet(x4)-containing-contigs-assembled-from-metagenomic-sequencing-data-with-plasmid.pdf'
citation: '<br/><b>Ding Y, Er S et al. (2024)</b>. Comparison of <i>tet</i>(X4)-containing contigs assembled from metagenomic sequencing data with plasmid sequences of isolates from a cohort of healthy subjects. <i>Microbiol Spectr12:e03969-23</i>. https://doi.org/10.1128/spectrum.03969-23'
# citation: 'Your Name, You. (2009). &quot;Paper Title Number 1.&quot; <i>Journal 1</i>. 1(1).'
---

Abstract
---
Recently discovered *tet*(X) gene variants have provided new insights into microbial antibiotic resistance mechanisms and their potential consequences for public health. This study focused on detection, analysis, and characterization of Tet(X4)-positive Enterobacterales from the gut microbiota of a healthy cohort of individuals in Singapore using cultivation-dependent and cultivation-independent approaches. Twelve Tet(X4)-positive Enterobacterales strains that were previously obtained from the cohort were fully genome-sequenced and comparatively analyzed. A metagenomic sequencing (MS) data set of the same samples was mined for contigs that harbored the *tet*(X4) resistance gene. The sequences of *tet*(X4)-containing contigs and plasmids sequences were compared. The presence of the resistance genes *floR* and *estT* (previously annotated as *catD*) was detected in the same cassette in 10 and 12 out of the 12 *tet*(X4)-carrying plasmids, respectively. MS detected *tet*(X4)-containing contigs in 2 out of the 109 subjects, while cultivation-dependent analysis previously reported a prevalence of 10.1%. The *tet*(X4)-containing sequences assembled from MS data are relatively short (~14 to 33 kb) but show high similarity to the respective plasmid sequences of the isolates. Our findings show that MS can complement efforts in the surveillance of antibiotic resistance genes for clinical samples, while it has a lower sensitivity than a cultivation-based method when the target organism has a low abundance. Further optimization is required if MS is to be utilized in antibiotic resistance surveillance.

Importance
---
The global rise in antibiotic resistance makes it necessary to develop and apply new approaches to detect and monitor the prevalence of antibiotic resistance genes in human populations. In this regard, of particular interest are resistances against last-resort antibiotics, such as tigecycline. In this study, we show that metagenomic sequencing can help to detect high abundance of the tigecycline resistance gene *tet*(X4) in fecal samples from a cohort of healthy human subjects. However, cultivation-based approaches currently remain the most reliable and cost-effective method for detection of antibiotic-resistant bacteria.